packages/@webex/internal-plugin-voicea/src/voicea.ts

```
@@ -501,6 +501,16 @@ export class VoiceaChannel extends WebexPlugin implements IVoiceaChannel {
       });
   };
 
+  /**
+   * In meeting Spoken Language changed event
+   * @param {string} languageCode
+   * @returns {void}
+   */
+  public onSpokenLanguageUpdate = (languageCode: string): void => {
+    // @ts-ignore
+    return this.trigger(EVENT_TRIGGERS.SPOKEN_LANGUAGE_UPDATE, {languageCode});
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)